File: src/main/java/scala/learn/akka/cto/MapActor.java
package scala.learn.akka.cto;

import akka.actor.ActorRef;
import akka.actor.UntypedActor;

/**
 * Created by Lei on 2016-8-8.
 */
public class MapActor  extends UntypedActor {
    public MapActor(ActorRef reductActor) {
        this.reductActor = reductActor;
    }

    private ActorRef reductActor;
    @Override
    public void onReceive(Object message) throws Exception {

    }
}

File: src/main/scala/scala/learn/akka/cto/MapActor.scala
package scala.learn.akka.cto

import akka.actor.Actor.Receive
import akka.actor.{Actor, UntypedActor}

/**
  * Created by Lei on 2016-8-11.
  */
class MapActor extends UntypedActor{

  override def onReceive(message: Any): Unit = {
    message match {
      case ms:String=>_
      case _=>unhandled(message)
    }
  }
}
